File: program/program/convert-seconds-to-days/ConvertSecondsToDays.java
import java.util.Scanner;  

public class ConvertSecondsToDays {
	
	public static int ConvertSectoDay(int seconds) {
		return seconds / 86400;
	}
	
	
	public static void main(String[] args) {
	    Scanner scanner = new Scanner(System.in); 
	    int seconds = scanner.nextInt();	 

		System.out.println(ConvertSectoDay(seconds));
	}

}

File: program/program/convert-seconds-to-days/ConvertSecondsToDays.kt
import java.util.Scanner

fun main() {
    print("Enter Seconds : ")
    println()
    var sec = readLine()

    try {
        println("$sec seconds to days : " + ConvertSecondsToDays(sec!!.toInt()))
    } catch (exception: Exception) {
        println("Input string was not in a correct format")
    }

}

private fun ConvertSecondsToDays(sec: Int): Int {
    val secToDays = sec / 86400

    return  secToDays
}